packages/plugin-core/assets/dendron-ws/tutorial/tutorial.4-images-and-diagrams.md

```diff
@@ -0,0 +1,78 @@
+---
+id: vAy0awBTwDaxKGHWUdSaP
+title: Images and Diagrams
+desc: ''
+updated: 1624452350303
+created: 1624352425213
+---
+After completing this section you'll know how to create rich content within your Markdown notes.
+
+### Images
+
+As you may have already noticed already in the tutorial pages, you can insert images into your notes. 
+
+> 🌱 The easiest way to do this is to first download the [Paste Image](https://marketplace.visualstudio.com/items?itemName=mushan.vscode-paste-image) extension. Once installed, copy the image onto your clipboard, and then use the `Paste Image` command into your editor pane.
```

Comment:
feel free to add it to our plugin bundle. see [here](https://github.com/dendronhq/dendron/blob/master/packages/plugin-core/package.json#L28:L28)